src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -416,7 +416,7 @@ ${styles()}
 // Utility helpers
 
 function formatCluster(cluster: any): string {
-    return cluster.resourceGroup + '/' + cluster.name;
+    return `${cluster.resourceGroup}'/'${cluster.name}`;
```

Comment:
Are you sure this is right?  It looks like it will introduce spurious characters e.g. `myrg/mycluster` -> `myrg'/'mycluster` - can you check it still works correctly?

Reply:
oh derp. good catch.